build.assets/Dockerfile

```diff
@@ -19,7 +19,26 @@ ENV LANGUAGE="en_US.UTF-8" \
 
 RUN apt-get update -y --fix-missing && \
     apt-get -q -y upgrade && \
-    apt-get install -q -y apt-utils curl gcc gcc-multilib git gzip libbpfcc-dev libc6-dev libpam-dev libsqlite3-0 locales make net-tools tar tree zip shellcheck && \
+    apt-get install -q -y \
+        apt-utils \
+        curl \
+        gcc \
+        gcc-multilib \
+        git \
+        gzip \
+        libbpfcc-dev \
+        libc6-dev \
+        libpam-dev \
+        libsqlite3-0 \
+        locales \
+        make \
+        mingw-w64 \
```

Comment:
Note to reviewers: this is the only new package added.
Everything else is formatting.